ykrt/src/compile/jitc_yk/jit_ir.rs

```diff
@@ -17,13 +17,10 @@
 //!  * `Inst`: "instruction"
 //!  * `Ty`: "type"
 //!
-//! To get human-readable representations of IR data structures, most of the IR constructs either:
-//!  * implement [std::fmt::Display] and thus can be stringified with (e.g.) `format!`, or
-//!  * if extra information is needed, have a method called `display()` which takes extra arguments
-//!    and returns something which does implement [std::fmt::Display].
-//!
-//! In the human readable IR, all names should be lowercase. Try to keep them short. Underscores
```

Comment:
I guess you thought this bit was obvious?

Reply:
Yes.